packages/fluentui/react-icons-northstar/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@fluentui/react-icons-northstar",
+  "description": "A set svg icons and utilities for creating custom ones.",
+  "version": "0.47.0",
+  "author": "Marija Najdova <mnajdova@gmail.com>",
```

Comment:
```suggestion
```

As I understand packages in Fluent doesn't have `author` field. @kenotron ?

Reply:
Yes let's please remove authors.

Reply:
A bunch of the other imported packages also have author fields (usually crediting Levi), which in general I think we should consider removing--at least to me it seems weird to give credit to one person when all this is actually a team effort.

Reply:
ok, will remove it. I just copied this from other package and changed it..